library/src/main/java/com/github/paolorotolo/appintro/AppIntro2Fragment.java

```diff
@@ -36,6 +75,20 @@ public static AppIntroFragment newInstance(CharSequence title, @FontRes int titl
                 0, 0);
     }
 
+    /**
+     * Generates new instance for AppIntro2Fragment
+     *
+     * @param title         CharSequence which will be the slide title
+     * @param titleTypeface CharSequence the URL of the custom typeface for
+     *                      the title found at assets folder
+     * @param description   CharSequence which will be the slide description
+     * @param descTypeface  CharSequence the URL of the custom typeface for
+     *                      the description found at assets folder
+     * @param imageDrawable @DrawableRes (Integer) the image that will be
+     *                      displayed, obtained from Resources
+     * @param bgColor       @ColorInt (Integer) custom background color
+     * @return AppIntro2Fragment created instance
+     */
     public static AppIntroFragment newInstance(CharSequence title, String titleTypeface,
```

Comment:
Ideally this should also be `@deprecated` and we should have a method with a `SliderPage` parameter only (similarly to `AppIntroFragment`).

Reply:
Applied on commit 0b83258